libs/dh/message-archive/feature-log-search/src/lib/dh-message-archive-log-search.component.html

```diff
@@ -16,35 +16,26 @@
 -->
 
 <ng-container *transloco="let transloco; read: 'messageArchive.search'">
-  <h1>{{ transloco('title') }}</h1>
+  <h1>{{ transloco("title") }}</h1>
   <form #searchForm name="log-search-form" (ngSubmit)="onSubmit()">
     <div class="wrapper">
       <watt-form-field size="normal">
         <watt-label class="watt-text-m">{{
-          transloco('documenttype')
+          transloco("documenttype")
         }}</watt-label>
 
-        <select
-          #searchInputDocumentType
-          wattInput
-          name="documentType"
+        <watt-dropdown
           [(ngModel)]="this.searchCriteria.rsmName"
+          [formControl]="rsmFormControl"
+          [options]="rsmFormFieldOptions"
+          [placeholder]="transloco('ChooseDocumentType')"
+          [noOptionsFoundLabel]="'-'"
```

Comment:
issue: the `noOptionsFoundLabel` only works with multi-select. Setting it on single selection dropdown will do nothing.